### PR TITLE
🚀 Add basic Deno (🚀) implementation 🚀🚀

### PR DESCRIPTION
**Pull request by satoqz**
TIFU by revealing my linux system to my family

Alright, so here’s the dealio.

I have a large Finnish family with many cousins, uncles, aunts, nephews, et cetera. Every once in awhile we’ll all gather up to celebrate the anniversary of our great grandparents’ wedding, a special occasion marking the creation of our family. They’re long gone now, but it’s the memory that counts, and we celebrate by cooking a feast for the ages, rivaling thanksgiving dinner.

Now upon the eve of this anniversary all “da boys” (as us men in the family affectionately call ourselves) gather up and go to a famous local bar in downtown Boston. They only accept cash, very “old school” feel to the place, but it makes for some great stories. Usually the place is a blast, with all sorts of banter and drunken shenanigans occurring, but not last night...

Now before I divulge the details, I ought to give some background as to how I fit into this family. Where my father and I are well integrated into the family, my mother is quite the outsider. She comes from wealthier, southern origins, around the mid-Atlantic part of the country. Because of this, she’s always viewed the family as “working class”, with her posh accent sticking out like a sore thumb at gatherings of Bostonians. By extension, I’ve always felt I had to prove to my family that I can “hang” with them so to speak, and in a way I worry that they see my mom as the woman who pulled my dad away from their tight knit circle.

Now let’s cut back to last night at the bar. Laughs, drinks, and stories all around the bar as we enjoy ourselves. My cousin Vinny invites me and some of the younger guys to play truth or drink. At this point I’m already quite drunk, but accept for the fun of it. A few questions in and Vinny asks me “what distro do you use? I know you’re packing a fancy rice Anon, right?”

Of course my other cousins all groan with embarrassment, but I foolishly answer instantly, unaware of the ramifications of my response.

“just ubuntu” I say too confidently, and suddenly all eyes are on me.

“Drink Anon, I know you’re bullshitting me man!” says Vinny, with a twinge of nervous angst in his voice.

“I’m serious Vin, that’s all I got, haha”

Dead silence. I gaze across the bar to see my entire family looking in shock and awe at my response. Vinny attempts to get everyone focused back on the game, and everyone resumes at a quieter tempo, but I knew that I had just majorly fucked up.

After leaving the bar, my uncle Paul pulls me aside, puts both hands on my shoulder and says “Kid, please tell me you’re just fucking around about your OS”

“No paul, Jesus what the fuck is your problem?” I say indignantly

“Anon, don’t you know everyone in this family is an elitist? For Christ's sakes I’m on the less computer talented side and still use my own OS i coded in x86 assembly.”

“So what? Why the hell does the OS matter to you so much?” I’m starting to get worried at this point, I’ve never seen Paul look at me with such intensity.

“Matter to me? It’s matters to the whole damn family! The OS is more than a string, it quantifies your entire personality. You can’t call yourself a member of this family if your walking around with a fucking windows installation on your laptop”

Paul went on explaining the history of this family, and how the men found success through leveraging their superior operating systems, both literally and figuratively. According to him, my dad was somewhat of a legend, with the most beautiful and efficient nixos config. Apparently during a final 200 meter dash in highschool, my father won by literally being so fast in vim he recoded the universe to make him first place. I always felt like I never lived up to my father’s athletic record, but now I know that I come short in more ways than one.

So how could my dad’s OS be so amazing, and mine be so shitty? Because everyone knows elitism is genetic, I had to investigate.

It turns that FOSS elitism correlates with genes on both the X and Y chromosomes, meaning that both my mother and father’s genetics are responsible. My father’s side obviously is known for their legendary rices, so I knew I had to speak with my mother.

She was incredibly reluctant to describe her male family members’ operating systems to me for some reason... odd. I knew I had to investigate further, so I called some of my cousins and uncles from that side. Turns out, the family has relatively average installs, but all better than mine. Combined they averaged on void and mint.

Here’s what makes no sense, if my dad’s side has the best OSs, and my mom’s side has an average, why don’t I fall within that range? I suspected my mother’s hesitancy to speak on the matter indicated a darker secret, so I went digging.

My mom was friends with a man she knew from law school named Brian, and I always suspected she might have had a thing for him. I reach out to talk, and he gladly accepts.

Over the phone, I ask him “So what operating system do you use?”

“My OS? Ahh, unfortunately I was born with a rather small brain, so i use Windows 11, but it gets the job done. It’s about average in my family”

Bingo, what lies in between Windows 11 and Void? Ubuntu. Brian was my father. But before I had a chance to ask him, the phone line disconnected.

My father stands across from me, holding the phone cord in his hand. He had been on the other line, and must have figured it out.

“I always knew...” he said with a solemn look in his eye. He sulked away and retreated to his room, feeling unworthy to face the rest of the family.

I realize now that I needed to at least make amends with my cousins, they might not be able to respect my distro on elitist levels, but maybe I could demonstrate the skill and speed of myself instead.

The dinner party was about to start, my mother and father had clearly been arguing and didn’t show up. Nobody bothered to look at me, and when they did catch my eyes I only saw disdain.

I realized now was the chance, and stood up on the table.

“I challenge anybody who’s man enough to a vim fight!” I declared, whipping my inferior installation out into the air for everyone to see.

At first I was met with silence, but then Vinny steps up, with rage in his eyes...

“Tough talk for a fella with an ubuntu install”

He joined me on the table, summoning his terrifying i3 arch installation. I quickly assumed a guard position, I was 3 levels of abstractions short from a fair fight, so I had to play defensively.

He charged with his M1 MacBook in hand, taping wildly, but clearly underestimating my maneuverability. I typed and landed 3 quick full alphabets with the tip of my little finger on my keyboard. He reposted quickly, but I chambered his advantage and managed to strike at the :w. Vinny was down, but soon the rest of the family whipped out their computers and prepared to engage in an all out melee.

The smell of fried thermal paste filled the air as men took their sides on the battlefield. Some came to defend me, admiring my valor in single combat with Vinny, while others came to avenge him. Soon a frontline emerged, reminiscent of the rust warfare of the early modern period. Men on the front pinged and rerouted the enemies to make ground, while occasionally allowing for volleys of dust to fly across the room as artillery.

The battle was intense, casualties began to stack up, and it looked like my side was going to collapse.

Then my father came downstairs, his legendary nixos rig in hand.

“You may not be my son, anon, but you’ll always be my boy”

He and my allies charged, rallying our fallen comrades. I’d never seen a man vimfight like my father before. He used his massive pc like a 90s hacker, cutting down two, three installs at once.

Soon the battle ended, with Vinny’s allies surrendering in defeat. The room was covered in broken motherboards and fan dust, thermal paste dripping from the walls and ceiling.

I don’t know where I stand with the family currently. Many have learned to accept me, but I’m sure it will take years before the rest consider it. Who knows how many more wars this family will fight before peace is had... and it’s all my fault.

TL;DR: I revealed my linux system to my family, which escalated into a massive war.

### CODE REVIEW

**Comment by tectrixer** [comment]
Looks great, thx for contributing 👍

**Review by tectrixer — APPROVED** [review]
